modules/virt-customizing-storage-profile.adoc

```diff
@@ -108,3 +108,24 @@ status:
 <1> Specify the access mode.
 <2> Specify the volume mode.
 <3> Specify the default cloning strategy.
+
+[id="virt-customizing-storage-profile-providers-and-default-behaviors_{context}"]
+=== Storage Providers and Default Behaviors
```

Comment:
Only a suggestion, but wondering if it this could be just the title of the table instead of its own distinct section?
```suggestion
.Storage providers and default behaviors
```
Feels maybe a little more natural, particularly if it's just a table without any lead in sentences. You could keep the ID as-is, main thing you'd lose is just being able to have the handy anchor-link selector for copy/paste in the rendered build.